arduino/smartcar/smartcar.ino

```diff
@@ -14,18 +18,77 @@ void setup() {
 }
 
 void loop() {
-  car.setSpeed(90);
-  preventCrash();
+  car.setSpeed(50); 
+  preventCrashWithUltrasonic(); 
+  preventCrashWithInfraredFront(); 
+  preventCrashWithInfraredLeft();
+  preventCrahsWithInfraredRight();
+  preventCrashWithInfraredBack();
```

Comment:
All these preventCrashWith... methods should be called from the more general-case preventCrash() method.

Reply:
Thanks. Added a more general method preventCrash()